src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -83,15 +77,41 @@
               Maximum 5 files
             </li>
           </ul>
+
+          <v-btn
+            id="add-document-button"
+            outlined
+            color="primary"
+            class="btn-outlined-primary mt-4"
+            :disabled="numFiles >= 5"
+            @click="onClickAddDocumentButton()"
+          >
+            <v-icon>mdi-plus</v-icon>
+            <span>Add a Document</span>
+          </v-btn>
+
+          <p
+            v-if="customErrorMessage"
+            class="error-text mt-4 mb-0"
+          >
+            {{ customErrorMessage }}
+          </p>
+
+          <FileUploadPreview
+            ref="fileUploadPreview"
+            class="d-none mt-6"
+            :maxSize="MAX_FILE_SIZE"
+            :customErrorMessage.sync="customErrorMessage"
+            @fileValidity="onFileValidity($event)"
+            @fileSelected="onFileSelected($event)"
+          />
```

Comment:
Note that component is hidden ("d-none") but still rendered.

Note also the 2-way sync with the customErrorMessage (which may be set by THIS component or or the file upload sub-component).